doc/README.md

```diff
@@ -1,3 +1,9 @@
+<!-- {ignore} -->
+## ⚠️ Warning:
+This is a source file for the docs site and may not be up to date with the latest version.
```

Comment:
is this supposed to show on github only?
how come is github out-of-date? if anything can be out of date it's the site...
I think we should support viewing in both platforms

Reply:
I think the disclaimer is valid, but the message is confusing. It should say that this source can contain unpublished features, or something like that

Reply:
if we get rid of develop, we can remove this message then right?

Reply:
Master will still be ahead of the last published version, so I think the message should say something like:
> This is a source file for the docs site and might contain documentation about features or changes that are not yet released.